.github/workflows/branch.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v4
         with:
-          go-version: 1.19
+          go-version: "1.20"
```

Comment:
Indeed there is. Without quotes, it is parsed as a decimal and then there is no difference between 1.20 and 1.2. So it would actually use version 1.2.